index.d.ts

```diff
@@ -24,7 +24,7 @@ declare namespace Moleculer {
 	type ActionParamSchema = { [key: string]: any };
 	type ActionParamTypes = "boolean" | "number" | "string" | "object" | "array" | ActionParamSchema;
 	type ActionParams = { [key: string]: ActionParamTypes };
-	type MetricsOptions = { params?: "boolean" | "array" | "function", meta?: "boolean" | "array" | "function" };
+	type MetricsOptions = { params?: boolean | string[] | function, meta?: boolean | string[] | function };
```

Comment:
`function` isn't a type, please use a function fingerprint to indicate input parameters and the output of the function.

Is the array only ever going to be an array of strings or could it be numbers or objects?

@icebob perhaps you could clarify the expected function parameters if any?

Reply:
The array can be only string array.
The function signature is `function(params: ActionParams)` or `function(meta: object)`

Reply:
The functions have no return type (meaning void)?

Reply:
Ohh sorry. But it can return `any` :)